.github/workflows/deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows: ["Publish Docker Image"]
+    types: ["completed"]
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    # If new docker image is published successfully
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Deploy
+        uses: appleboy/ssh-action@v0.1.5
+        with:
+          host: ${{ secret.SSH_HOST }}
+          username: ${{ secret.SSH_USER }}
+          key: ${{ secret.SSH_KEY }}
```

Comment:
It seems that all of those secrets are not set up in the repository secrets yet. Should they be configured before the merge?

Reply:
@JacobLinCool done 👍